### PR TITLE
follow@alpha 0.0.1-alpha.22 (new cask)

### PR DESCRIPTION
**Pull request by realsunyz**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/f/follow@alpha.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  url "https://github.com/RSSNext/Follow/releases/download/v#{version}/Follow-#{version}-macos-universal.dmg",
+      verified: "github.com/RSSNext/Follow/"
+  name "Follow"
+  desc "Information browser"
+  homepage "https://follow.is/"
+
+  livecheck do
+    url :url
+    regex(/^v?(\d+(?:\.\d+)+(?:[._-]alpha[._-]?\d+)?)$/i)
```
Comment (reitermarkus) [inline Casks/f/follow@alpha.rb:13]: `strategy :git` should work for this.

Comment (realSunyz) [reply Casks/f/follow@alpha.rb:13]: But neither [follow.rb](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/f/follow.rb) nor [follow@nightly.rb](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/f/follow%40nightly.rb) use `strategy :git`

**Comment by realSunyz** [comment]
I will add conflicting casks for `follow` and `follow@nightly` after this PR is merged.

**Review by bevanjkay — APPROVED** [review]

**Comment by khipp** [comment]
It seems like upstream promoted their releases from alpha to beta when moving to the first minor release. Do we know whether upstream will continue to release alpha versions, or are they moving towards stable releases in addition to their nightly versions?

**Comment by realSunyz** [comment]
I see that they released both `v0.0.1-alpha.22` and `v0.1.0-beta.0` yesterday, and there is no sign that this will be the last alpha release. The nightly versions are automatically released every day, so I think they will probably continue to maintain all three versions.

**Pull request merged**